### ISSUE TITLE
Need image caption right under image not at bottom of screen

### ISSUE THREAD

**Issue by ProvisionInc** [fancyapps/fancybox#2486]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by acwolff** [comment]
You mean like this: https://andrewolff.nl/FotoSerie/Stadswandelingen/Almelo/index.htm#S-14 ?

**Comment by ProvisionInc** [comment]
Yes. I upgraded from ver 1 of FancyBox where I had the caption right under each image. Now with version 3 it seems the developer has gone backward.  I can't figure out how to get the caption right under the image. It is stuck way down at the bottom of the screen. Can you tell me how to correct this in Fancybox 3?

**Comment by ProvisionInc** [comment]
Very cool images!  Did you take the pictures?

**Comment by acwolff** [comment]
Yes I took all the pictures displayed on my website.

If you like to investigate the code you better look atr this test-album: https://andrewolff.jalbum.net/Vecht_Beerze_FB/
